client/landing/stepper/declarative-flow/internals/steps-repository/components/flow-card/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { Flex, FlexBlock, FlexItem, Card, CardBody, Icon } from '@wordpress/components';
+import { chevronRight } from '@wordpress/icons';
+import './style.scss';
+
+interface FlowCardProps {
+	icon: JSX.Element;
+	onClick: () => void;
+	text: string;
+	title: string;
+}
+
+const FlowCard = ( { icon, onClick, text, title }: FlowCardProps ) => {
+	return (
+		<Card className="flow-question" as="button" size="small" onClick={ onClick }>
+			<CardBody>
+				<Flex>
+					<FlexItem className="flow-question__icon">
+						<Icon icon={ icon } size={ 24 } />
+					</FlexItem>
+					<FlexBlock>
+						<h3 className="flow-question__heading">{ title }</h3>
+						<p className="flow-question__description">{ text }</p>
+					</FlexBlock>
+					<FlexItem>
+						<Icon icon={ chevronRight } size={ 24 } />
```

Comment:
Need to check screen readers don't see this icon.

Reply:
I did just test this screen with tabbing and screen reader voice over. It works well. The icon isn't recognized for purposes of tabbing and navigation.